routes/login.js

```diff
@@ -0,0 +1,99 @@
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const config = require("../config");
+const jwtService = require("../services/JwtService");
+
+const userService = require("../services/UserService");
+
+router.get("/", (req, res) => {
+    userService
+        .getAllUsers()
+        .then((users) => {
+            res.json({ users: users });
+        })
+        .catch(() => {
+            res.json({ msg: "no user" });
+        });
+});
+
+router.get("/verify", (req, res) => {
+    const user = jwtService.verify(req.headers["authorization"]);
+    user != undefined
+        ? res.status(200).json({ msg: "verified" })
+        : res.status(401).json({ msg: "unauthorized" });
+});
+
+router.post("/login", (req, res) => {
+    const { zprn, password } = req.body;
+    if (zprn === undefined || password === undefined) {
+        res.status(401).json({ msg: "unauthorized" });
+    } else {
+        userService
+            .getUser(zprn)
+            .then((user) => {
+                if (user != undefined) {
+                    bcrypt.compare(password, user.password, (error, match) => {
+                        if (error) {
+                            res.status(500).json({ msg: "Error occured" });
```

Comment:
I think we should have a `model` for response errors, so that they are consistent everywhere. We can have a error object which will have status `code` and a error `msg`?
Also if we need in the future we can easily add new data to that object, without drastically having to change code everywhere.